website/docs/plugins/ax_sweeper.md

```diff
@@ -85,6 +85,8 @@ In general, the plugin supports setting all the Ax supported [Parameters](https:
 
 Note that if you want to sample integers in the range `-5` to `5`, you need to specify the range as `int(interval(-5, 5))` (in the command line) or `[-5, 5]` (in config). If you want to sample floats in range `-5` to `5`, you need to specify the range as `interval(-5, 5)` (in the command line) or `[-5.0, 5.0]` (in config).
 
+Generally, Ax expects the function being optimized to return a tuple of `(measurement_value, measurement_uncertainty)`. If the return value is instead a scalar, Ax by default treats the uncertainty as an unknown and infers its value as the experiment progresses. The example above uses an exact function, so the `is_errorless` parameter is set to `True`. As a result, the uncertainty of each scalar measurement is set to 0. Note that returning a `(measurement_value, measurement_uncertainty)` tuple will override this behavior.
+
```

Comment:
Maybe something like:
```
The Ax Sweeper assumes the optimized function is a noisy function with unknown measurement uncertainty.
This can be changed by overriding the `is_errorless` parameter to True, which is equivalent to a unction that is always certain about it's measurements.

Functions that can provide actual  measurement uncertainty can do so by returning the tuple `(measurement_value, measurement_uncertainty)` instead of a scalar value.
```

I also think we need to do a better job at describing what the measurement_uncertainty actually is and how to compute it. Is this [SEM (Standard deviation)](https://en.wikipedia.org/wiki/Standard_error)?

Reply:
Yes, SEM is as described in that article. Also I like your rephrasing.